static/app/views/settings/project/navigationConfiguration.tsx

```diff
@@ -64,6 +64,11 @@ export default function getConfiguration({
           title: t('User Feedback'),
           show: () => !isSelfHostedErrorsOnly,
         },
+        {
+          path: `${pathPrefix}/toolbar/`,
+          title: t('Developer Toolbar'),
+          show: () => !!organization?.features?.includes('dev-toolbar-ui'),
```

Comment:
Because of the `?`'s this statement might return undefined. 
> in js !! will test the truthiness of any value and convert it to a boolean. Similar to how Boolean(var) works.

Is this still correct though?